gradle/dependencies.gradle

```diff
@@ -1,7 +1,7 @@
 def versions = [
     apolloVersion                : '0.3.2-SNAPSHOT',
     cacheVersion                 : '2.0.2',
-    okHttpVersion                : '3.6.0',
+    okHttpVersion                : '3.8.1',
```

Comment:
alternatively, i can explicitly require Okio 1.13.0 if there's a reason to stay on OkHttp 3.6.0.